src/accordion/accordion.js

```diff
@@ -82,7 +82,7 @@ angular.module('ui.bootstrap.accordion', ['ui.bootstrap.collapse'])
       scope.openClass = attrs.openClass || 'panel-open';
       scope.panelClass = attrs.panelClass;
       scope.$watch('isOpen', function(value) {
-        element.toggleClass(scope.openClass, value);
+        element.toggleClass(scope.openClass, Boolean(value));
```

Comment:
I would change this to `!!value` - the reason is because the `Boolean` constructor can be overridden by the user, whereas using `!!` to force to a Boolean cannot be overridden.